### DISCUSSION TITLE
Feedback for Eddie Jaoude's free GitHub profile course

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#2778] [Show and tell]
Free course link https://www.eddiejaoude.io/course-github-profile-landing

Please provide your thoughts, feedback and ideas below 👇

**Comment by iason9** [comment]
I have some points:
- if you could resize and make smaller the main left image of you because it is a bit distracting when watching the videos.

- the section about GitHub actions need a practical example of finding the action and putting it into the Yaml file

- maybe put the top menu in the left menu and make the inner sections as dropdowns, it is a bit confusing having two menus

- Other than that the course is very professional and well built i really loved the progress bar and didn't find bugs in it it works well when you go through a section and the certificates on the end are nice

- You have a really good base to build more courses now and hope you do!

**Reply by eddiejaoude** [reply]
Thank you so much for taking the course Iason and giving the constructive feedback ❤️  Yes more courses to come! I have already started the next one 🤓

**Reply by iason9** [reply]
Great News! glad i could give some suggestions that might help, Good luck with the next course I am sure it will awesome as well!
